### ISSUE TITLE
Loans with interest

### ISSUE THREAD

**Issue by eX00r** [powerpaul17/nc_money#32]
Hello,

thanks for your wonderful work, this is a huge step forward in moving all our business processes to nextcloud!

It would be great to be able to add entries for loans with automatic calculation of interest for their term. This would mean for each entry users would be requested to enter the amount, the interest rate, the term etc.. Calculation should then be automatically done for each recurring term.

Do you think that's feasible?

best regards

**Comment by powerpaul17** [comment]
What do you mean by "automatically"? In Gnucash for example there is a tool to calculate this but you would have to enter the corresponding transactions manually. Do you mean some kind of scheduled transaction which calculates interests and such?

**Comment by eX00r** [comment]
Yes exactly. I am thinking of something like the loan payoff simulator in You Need A Budget which at one hand allows you to open an account for each loan with the relevant details like Amount, Interest Rate, Target Payoff Date and/or Monthly Payoff Amount. And from there allows you to automatically schedule the transactions in a specific budgeting category needed for the future to meet your plan.

![image](https://user-images.githubusercontent.com/10723825/215051711-749335f6-0124-4f18-aead-d3f1e27e03e6.png)
![image](https://user-images.githubusercontent.com/10723825/215051963-6bdd3b05-ab4b-45de-a420-4f384150051a.png)

**Comment by MrLoverLoverMMMM** [comment]
I need that as well :)
